_includes/pbc/all/install-features/202404.0/install-dynamic-multistore.md

```diff
@@ -195,15 +180,37 @@ docker:
 In this configuration, region is used for entities like services, endpoints, or applications. The `SPRYKER_DYNAMIC_STORE_MODE`  variable enables the dynamic multistore feature. Make sure store is not used in the new configuration to avoid deployment failures.
 
 
-3. Add the following configuration:
+
+### Set up configuration
+
+Before the introduction of dynamic multistore, configuration was managed in `config/Shared/stores.php`. With dynamic multistore, configuration is managed in the database. `config/Shared/stores.php` and `config/Shared/default_store.php` are now obsolete.
+
+The default configuration is imported using data import modules, such as StoreDataImport, LocaleDataImport, CountryDataImport, and other feature-related imports, like CurrencyDataImport.
+
+Take the following steps to set up configuration.
+
+#### Define the region stores context by domain
+
+With dynamic multistore, you can define region or store using domains or headers. We recommend defining region using domains.
+
+{% info_block infoBox "Changing the domain name" %}
+
+We recommend making `de.mysprykershop.com` a mirror of `eu.mysprykershop.com` to preserve the availability of old links in search engines.
+
+{% endinfo_block %}
+
+
+#### Configure the application
+
+1. Add the following configuration:
 
 | CONFIGURATION        | SPECIFICATION | NAMESPACE |
 |----------------------|---------------| --- |
-| Default RabbitMQ connection: `config/Shared/config_default.php`. | Enables the connection for queues to be set dynamically. Use the `SPRYKER_CURRENT_REGION` environment variable to set the configuration for queues. |  |
+|  Default RabbitMQ connection: `config/Shared/config_default.php`. | Enables the connection for queues to be set dynamically. Use the `SPRYKER_CURRENT_REGION` environment variable to set the configuration for queues. |  |
```

Comment:
What are we trying to explain by `Default RabbitMQ connection`?

Reply:
Default connection configuration is used in rabbit-mq module for getting default queue channel.